tests/test_x509_ext.py

```diff
@@ -112,3 +129,94 @@ def test_unsupported_extension(self, backend):
         )
         extensions = cert.extensions
         assert len(extensions) == 0
+
+
+@pytest.mark.requires_backend_interface(interface=RSABackend)
+@pytest.mark.requires_backend_interface(interface=X509Backend)
+class TestRSABasicConstraintsExtension(object):
```

Comment:
That's the only relation. We need to keep the `RSABackend` mark but we could remove the RSA component of the name.